train.py

```diff
@@ -25,14 +23,20 @@
 num_features = context_embeddings.shape[2]
 raw_question = questions[0][0]['question']
 
-words_in_context = tf.placeholder(tf.float32, [time_steps, batch_size, num_features])
```

Comment:
Are you sure about the dimensions? According to the [Keras API of LSTM](https://www.tensorflow.org/versions/r1.14/api_docs/python/tf/keras/layers/LSTM), the default value of the `time_major` parameter is false and so, this layer takes inputs in batch major mode. Moreover, this is consistent with the `tf.losses.softmax_cross_entropy` function.

Reply:
Good point, although here this line was removed. In the latest commit I just pushed, everything has been set to be `batch_size` major consistently, i.e. `[batch_size, time_steps, num_features]`